modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java

```diff
@@ -877,13 +876,13 @@ private void _assertContainsBinary(
 		Pattern pattern = Pattern.compile(
 			String.format("/%s/%d/\\d+\\.\\d+$", className, classPK));
 
-		Stream<String> entriesStream = entries.stream();
+		entries = ListUtil.filter(entries, pattern.asPredicate());
 
-		Assert.assertTrue(
+		Assert.assertFalse(
 			String.format(
 				"%s does not contain a binary entry for %s with primary key %s",
 				entries.toString(), className, classPK),
-			entriesStream.anyMatch(pattern.asPredicate()));
+			entries.isEmpty());
```

Comment:
@JanisZhang You can inline these to something like `Assert.assertTrue(..., ListUtil.isNotEmpty(ListUtil.filter(entries, predicate)))`.